rvv-intrinsic-rfc.md

```diff
@@ -246,6 +249,31 @@ size_t vsetvlmax_e8m8 ();
 
 There is no need to specify the behavior of tail and masked-off elements being undisturbed or agnostic. The default setting is tail agnostic and masked-off undisturbed. If users do not want to keep the values in masked-off elements, they could pass `vundefined()` as the `maskedoff` value.
 
+`SEW` and `LMUL` are a part of the naming. They are static information for the
+intrinsics.
+
+All of the intrinsic function having a `vl` argument to specify the active
+vector length, except few functions are work with fixed length, e.g. `vmv.x.s`,
+`vfmv.f.s`, `vundefined`, `vreinterpret`, `vlmul_ext`, `vlmul_trunc`, `vget`,
+`vset` and `vcreate`.
+
+The intrinsic function will only operate at most `VLMAX` element if `vl`
+argument are larger than `VLMAX`.
+
+The behavior is equivalence for following code, but we strongly suggesting the
```

Comment:
The semantics of following two snippets are equivalent. We strongly suggest the first form.